File: src/main/java/org/artembogomolova/demo/webapp/dao/IPhysicalAddressRepository.java
package org.artembogomolova.demo.webapp.dao;

import org.artembogomolova.demo.webapp.model.core.PhysicalAddress;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface IPhysicalAddressRepository extends CrudRepository<PhysicalAddress, Long> {

}

File: app/src/main/kotlin/org/artembogomolova/demo/webapp/main/dao/repo/IPhysicalAddressRepository.kt
package org.artembogomolova.demo.webapp.main.dao.repo

import org.artembogomolova.demo.webapp.main.domain.core.PhysicalAddress
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface IPhysicalAddressRepository : CrudRepository<PhysicalAddress?, Long?>